src/components/Checkout/Checkout.js

```diff
@@ -0,0 +1,936 @@
+                      />
+                    </FormGroup>
+                  </FormControl>
+                  <Typography className={classes.stepNum}>
+                    05
+                    <span className={classes.stepSlash}>/</span>
+                    <span className={classes.stepLabel}>Billing Address</span>
+                  </Typography>
+                  <Grid container spacing={2} className={classes.fieldRow}>
+                    <Grid item xs={12}>
+                      <CheckoutTextField
+                        placeholder="Address Line 1"
+                        name="address line 1"
+                        value={profile['address line 1']}
+                        onChange={handleChange}
+                        disabled={selectedProfile !== '' && !editing}
+                        valid={valid['address line 1']}
+                        validate={validateFilled}
+                        setValid={handleValidation}
+                        updateErrorDisplay={resetErrorDisplay}
+                      />
+                    </Grid>
+                    <Grid item xs={12}>
+                      <CheckoutTextField
+                        placeholder="Address Line 2"
+                        name="address line 2"
+                        value={profile['address line 2']}
+                        onChange={handleChange}
+                        disabled={selectedProfile !== '' && !editing}
+                        valid={valid['address line 2']}
+                        setValid={handleValidation}
+                        updateErrorDisplay={resetErrorDisplay}
+                      />
+                    </Grid>
+                    <Grid item xs={6}>
+                      <CheckoutTextField
+                        placeholder="City"
+                        name="city"
+                        value={profile.city}
+                        onChange={handleChange}
+                        disabled={selectedProfile !== '' && !editing}
+                        valid={valid.city}
+                        validate={validateFilled}
+                        setValid={handleValidation}
+                        updateErrorDisplay={resetErrorDisplay}
+                      />
+                    </Grid>
+                    <Grid item xs={6}>
+                      <CheckoutTextField
+                        placeholder="State"
+                        name="state"
+                        value={profile.state}
+                        onChange={handleChange}
+                        disabled={selectedProfile !== '' && !editing}
+                        valid={valid.state}
+                        validate={validateFilled}
+                        setValid={handleValidation}
+                        updateErrorDisplay={resetErrorDisplay}
+                      />
+                    </Grid>
+                    <Grid item xs={6}>
+                      <CheckoutTextField
```

Comment:
For the zipcode, do we want it to be an incremental input field?

Reply:
No, I was dealing with some issues with submitting the zipcode directly to the airtable as a string and everything was a lot cleaner if I just set the type to number. I can look for other methods to do this without the incremental input!